### DISCUSSION TITLE
How can I generally erase VPNHOOD from the server?

### DISCUSSION THREAD

**Discussion by mahdisarlak** [vpnhood/VpnHood#165] [Q&A] [answered]
How can I generally erase VPNHOOD from the server?

I installed VPNHOOD Server on my PC for testing, now VPNHOOD ClINET doesn't work on my computer?
What should I do ?

**Comment by bobvhood** [comment, marked as answer]
Hi, see the following page I created for you:
https://github.com/vpnhood/VpnHood/wiki/How-to-uninstall-VpnHood-Server-from-Linux

Let me know if there is any issue in it
